SimpleWAWS/Code/ResourcesManager.cs

```diff
@@ -269,7 +269,7 @@ private void DeleteResourceGroup(ResourceGroup resourceGroup)
                         //Enable ZRay
                         await site.EnableZRay(resourceGroup.GeoRegion);
                     }
-                    else if (((template.Language!=null) && template.Language.Equals("NodeJs", StringComparison.OrdinalIgnoreCase))|| template.Name.Equals("ExpressJs", StringComparison.OrdinalIgnoreCase))
+                    else if (((template.Language!=null) && template.Language.Equals("NodeJs", StringComparison.OrdinalIgnoreCase))|| template.Name.Equals("Express", StringComparison.OrdinalIgnoreCase))
```

Comment:
I know this is not related to this PR, but this could probably be simplified to 
```csharp
else if (template.Language?.Equals("NodeJs", StringComparison.OrdinalIgnoreCase) == true ||
    template.Name.Equals("Express", StringComparison.OrdinalIgnoreCase))
{
}
```